docs/descriptors/php_php_cs_fixer.md

```diff
@@ -14,7 +14,7 @@ PHP-CS-Fixer is able to fix rule violations detected, and even those marked as (
 
 ## php-cs-fixer documentation
 
-- Version in MegaLinter: **3.68.0**
+- Version in MegaLinter: **7.4.0**
```

Comment:
@nvuillam As I've already mentionned https://github.com/oxsecurity/megalinter/issues/4351#issuecomment-2599045558 there is a problem with automation to detect PHP-CS-Fixer : latest version available is 3.68.1 
7.4.0 correspond to PHP minimum requirement (see message below)

Reply:
@llaville default regex to extract version is `self.version_extract_regex = r"\d+(\.\d+)+"`

You can override it by defining version_extract_regex in phpcd-fixer descriptor :)